### DISCUSSION TITLE
Get battery level/percentage of crazyflie drone on python

### DISCUSSION THREAD

**Discussion by rithwikjayanth233** [bitcraze/discussions#855] [Q&A]
Hi,
I am running a script for my crazyflie drone and was wondering how I can get the battery level of the drone in regular time intervals. I viewed https://github.com/bitcraze/crazyflie-firmware/blob/master/src/hal/src/pm_stm32f4.c#L395 but this was fuzzy to understand. Is there any command/package I can use to get the battery level of the drone?

**Comment by knmcguire** [comment]
Hi!

With the python library you can monitor this [log variable](https://github.com/bitcraze/crazyflie-firmware/blob/e56b7fddf6e6ac9e10d9bacfffb4c4bfcd629219/src/hal/src/pm_stm32f4.c#L546) or perhaps any of [these values](https://www.bitcraze.io/documentation/repository/crazyflie-firmware/master/api/logs/#pm).

This step by step guide should tell you how to setup logging from the python script https://www.bitcraze.io/documentation/repository/crazyflie-lib-python/master/user-guides/sbs_connect_log_param/
